solc/CommandLineParser.cpp

```diff
@@ -1092,6 +1099,12 @@ General Information)").c_str(),
 		m_options.modelChecker.settings.contracts = move(*contracts);
 	}
 
+	if (m_args.count(g_strModelCheckerDivModSlacks))
+	{
+		bool divModSlacks = m_args[g_strModelCheckerDivModSlacks].as<bool>();
+		m_options.modelChecker.settings.divModWithSlacks = divModSlacks;
+	}
+
```

Comment:
I'm fine either way, just not sure how to code it, I'll check the existing code. If that's the case I should also change the `--model-checker-show-unproved true` to that

Reply:
It's enough to just remove the whole `po::value<bool>()->default_value(true),` part. It will become a boolean option and you can then check with `m_args.count()` if it was provided or not.

Reply:
Ah ok, makes sense, thanks!